app/src/openxr/cpp/OpenXRInputSource.h

```diff
@@ -63,6 +65,11 @@ class OpenXRInputSource {
     bool squeezeActionStarted { false };
     std::vector<float> axesContainer;
     crow::ElbowModelPtr elbow;
+    XrHandTrackerEXT mHandTracker { XR_NULL_HANDLE };
+    std::array<XrHandJointLocationEXT, XR_HAND_JOINT_COUNT_EXT> mHandJoints;
+    bool hasHandJoints { false };
+    XrHandTrackingAimStateFB mAimState { XR_TYPE_HAND_TRACKING_AIM_STATE_FB };
+    bool hasAimState { false };
```

Comment:
I know there are inconsistencies in this very same file, but let's use `mHasHandJoints` and `mHasAimState` instead

Reply:
Yes, though about it initially but chose to be consistent with existing code. Will change that as requested.